src/BinaryDefense.FSharp.Analyzers.Hashing/Library.fs

```diff
@@ -151,9 +151,9 @@ module Hashing =
 
     let matchers =
         [
-            "System.Security.Cryptography.MD5CryptoServiceProvider", MD5
-            "System.Security.Cryptography.MD5.Create", MD5
-            "System.Security.Cryptography.SHA1CryptoServiceProvider", SHA1
+            typedefof<MD5CryptoServiceProvider>.FullName, MD5
```

Comment:
nitpick: typedefof gets open-generic versions of the type passed in. for 'closed' types or types with no generics you can just used `typeof>.

Reply:
I always get this confused.